unlock-app/src/components/interface/locks/List/elements/EventCard.tsx

```diff
@@ -0,0 +1,146 @@
+import { FiArrowRight as ArrowRightIcon } from 'react-icons/fi'
+import { AiOutlineTag as TagIcon } from 'react-icons/ai'
+import { IoMdTime as TimeIcon } from 'react-icons/io'
+import { TbUsers as AttendeesIcon } from 'react-icons/tb'
+import Link from 'next/link'
+import { Card, Detail, Icon } from '@unlock-protocol/ui'
+import { CryptoIcon } from '@unlock-protocol/crypto-icon'
+import { PriceFormatter } from '@unlock-protocol/ui'
+import { WrappedAddress } from '~/components/interface/WrappedAddress'
+import dayjs from 'dayjs'
+import { useCheckoutConfigsByUserAndLock } from '~/hooks/useCheckoutConfig'
+import { useEventAttendees } from '~/hooks/useEventAttendees'
+import { useLockData } from '~/hooks/useLockData'
+
+interface EventCardProps {
+  event: {
+    name: string
+    lockAddress: string
+    network: number
+    slug: string
+    ticket: {
+      event_start_date: string
+    }
+    image: string
+  }
+}
+
+interface EventImageProps {
+  imageUrl: string
+}
+
+const EventImage = ({ imageUrl }: EventImageProps) => {
+  return (
+    <div className="relative block overflow-hidden rounded-2xl h-20 w-20 group">
+      <img alt="logo" className="object-cover h-full aspect-1" src={imageUrl} />
+    </div>
+  )
+}
+
+export const EventCard = ({ event }: EventCardProps) => {
+  const { name, lockAddress, network, slug, ticket, image: imageUrl } = event
```

Comment:
The event should have a checkout config and we should use that one?

Reply:
alright!